app/components/status-badge.jsx

```diff
@@ -0,0 +1,66 @@
+// https://github.com/EnCiv/civil-pursuit/issues/47
+
+'use strict'
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import cx from 'classnames';
+
+function StatusBadge(props) {
+    const { style = {}, className = "", name = "", status = "", number = undefined, ...otherProps } = props;
+    const classes = useStyles();
+
+    const statusMapping = {
+        progress: "Unread",
+        complete: "Complete",
+        inactive: "Group, Inactive",
+        error: "Most Important"
+    }
+
+    return (
+        <span className={cx(classes.container, classes[status.toLowerCase()])}>
+            <span className={classes.status}>{statusMapping[status.toLowerCase()]}</span>
+            {
+                number !== undefined && number !== null && (
+                    <span className={classes.number}>{Number(number)}</span>
+                )}
+        </span>
+    )
+}
+
+const useStyles = createUseStyles(theme => ({
+
+    container: {
+        borderRadius: '15px',
```

Comment:
Please use 1rem instead of 15px.  Consider that on some screens 1 rem is 15 pixels, on other screens it could be 8 pixels. If we define it in rem, it looks the same on both screens.

For boarderWidth 1px is okay per the project style guide, but I'm seeing that since this comes up so much, lets create a theme value for it. 
theme.border.width.thin and make it 0.0625rem.